query_tests.py

```diff
@@ -116,9 +116,55 @@ def test_return_height(platform="MK"):
     return util.EXIT_SUCCESS
 
 
+def test_get_service_name(platform="MK"):
+    # generate the filter code
+    result = generate_filter("get_service_name.cql", [])
+    assert result == util.EXIT_SUCCESS
+
+    # bootstrap the filter
+    storage_proc = bootstrap()
+
+    # first request
+    log.info("Sending request #1")
+    requests.send_request(platform)
+    storage_content = storage.query_storage()
+    text = storage_content.text
+    result_set = process_response(text)
+    assert "productpage-v1" in result_set, (
+        "expected productpage-v1 received %s" % result_set)
+
+    storage.kill_storage_mon(storage_proc)
+    log.info("get_service_name test succeeded.")
+    return util.EXIT_SUCCESS
+
+
+def test_request_size(platform="MK"):
+    # generate the filter code
+    result = generate_filter("request_size.cql", [])
+    assert result == util.EXIT_SUCCESS
+
+    # bootstrap the filter
+    storage_proc = bootstrap()
+
+    # first request
+    log.info("Sending request #1")
+    requests.send_request(platform)
+    storage_content = storage.query_storage()
+    text = storage_content.text
+    result_set = process_response(text)
+    assert "productpage-v1" in result_set, (
```

Comment:
It looks like request_size.cql still returns the size of the request at productpage, but wouldn't have productpage itself within the response.  Why is productpage-v1 in the result set?

Reply:
The test does not work yet, so I do not know what to check for. The most likely solution is to parse the outputs as integers and then check whether they are in a certain range.

Reply:
Okay, we should either put that range in, or put a comment to that effect.  Same thing for height, which should work with what we have in the compiler now.